crates/storage/provider/src/test_utils/mock.rs

```diff
@@ -341,7 +341,7 @@ impl<T: SignedTransaction, ChainSpec: EthChainSpec> TransactionsProvider
                         excess_blob_gas: block.header.excess_blob_gas,
                         timestamp: block.header.timestamp,
                     };
-                    return Ok(Some((tx.clone(), meta)))
+                    return Ok(Some((tx.clone(), meta)));
```

Comment:
```suggestion
                    return Ok(Some((tx.clone(), meta)))
```
pls revert all these new semicolons you can see in the diff (more places than here). take care to use only nightly toolchain for formatting reth, i.e. `cargo +nightly fmt`

Reply:
Fixed it & also rebased.  

I’d used `fmt` before, but not the `nightly` version. Now with the `nightly` version when a semicolon gets added to the line (my IDE added it automatically here), `fmt` doesn’t remove it. That’s why it’s in the commit.